pkgs/servers/xteve/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+let
+  owner = "xteve-project";
+  repo  = "xTeVe";
+  version = "2.2.0.200";
+
+in buildGoModule rec {
+  pname = "xteve";
+  inherit version;
+
+  src = fetchFromGitHub {
+    inherit owner repo;
+    rev = version;
+    sha256 = "sha256-hD4GudSkGZO41nR/CgcMg/SqKjpAO1yJDkfwa8AUges=";
+  };
+
+  vendorSha256 = "sha256-oPkSWpqNozfSFLIFsJ+e2pOL6CcR91YHbqibEVF2aSk=";
+
+  meta = with lib; {
+    description = ''
+      M3U Proxy for Plex DVR and Emby Live TV.
+    '';
+    homepage = "https://github.com/xteve-project/xTeVe";
+    license = licenses.mit;
+    maintainers = with maintainers; [ ];
```

Comment:
Usually we require that you (aka the package creator) list yourself as maintainer here.

However, I will suppose you are not interested on take this role. 
In this case, for the sake of completeness, please add myself (`AndersonTorres`) as maintainer.